setup.py

```diff
@@ -46,6 +46,13 @@ def read_md(filename):
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Framework :: Django',
+        'Framework :: Django :: 3',
+        'Framework :: Django :: 3.0',
+        'Framework :: Django :: 3.1',
+        'Framework :: Django :: 3.2',
+        'Framework :: Django :: 4',
+        'Framework :: Django :: 4.0',
```

Comment:
I was't sure about the Django 2.x support, so I left that out.

Reply:
Yes, Django 2.2 is supported, although I believe the EOL is tomorrow.

Reply:
Should I add it anyway?

Reply:
That would be fine.

Reply:
Done.